packages/card/stories/Card.stories.tsx

```diff
@@ -1,20 +1,18 @@
 import * as React from "react";
 import { ButtonCard, Card, LinkCard } from "../index";
 import { Story, Meta } from "@storybook/react";
-
-const sizes = ["xxs", "xs", "s", "m", "l", "xl", "xxl"];
+import { spacingSizeValues } from "../../storybookHelpers/controlConstants";
 
 export default {
   title: "Layout/Card",
   component: Card,
   subcomponents: { ButtonCard, LinkCard },
   argTypes: {
```

Comment:
I noticed there was a story for a Card with an `AspectRatio`, but the control for AspectRatio automatically defaults to an object. This seems to be the Storybook control type for arrays. I wasn't able to figure out how to get the control working so you could specify an aspect ratio. Do you know how it would be done, and if not should we disable the control?

Reply:
Good point. Now that I know about the mapping I may be able to do something about this to make this control more usable. Otherwise, I may disable it, it's a prop that probably is not used often.